lib/stack_master/command.rb

```diff
@@ -27,6 +27,12 @@ def perform
       end
     end
 
+    def initialize(config, stack_definition, options = Commander::Command::Options.new)
+      @config = config
+      @stack_definition = stack_definition
+      @options = options
+    end
```

Comment:
Pull up `initialize` method into `Command` module. Most of the commands initialize with these params. There are a few that differ though:

https://github.com/envato/stack_master/blob/d598e12de421ac7510e0e1bf1dd2e46d5bb88c91/lib/stack_master/commands/delete.rb#L7-L11

https://github.com/envato/stack_master/blob/d598e12de421ac7510e0e1bf1dd2e46d5bb88c91/lib/stack_master/commands/init.rb#L8-L12

https://github.com/envato/stack_master/blob/d598e12de421ac7510e0e1bf1dd2e46d5bb88c91/lib/stack_master/commands/list_stacks.rb#L10-L12

https://github.com/envato/stack_master/blob/d598e12de421ac7510e0e1bf1dd2e46d5bb88c91/lib/stack_master/commands/status.rb#L10-L13

These are fine to override the default initializer.